2020-11-03 17:55:53.302 - [Info] MITM Version: 1.26
2020-11-03 17:55:53.306 - [Info] Auto Access Enabled: true
2020-11-03 17:55:53.306 - [Debug] [Init] Auto Access Barrier: true
2020-11-03 17:55:53.321 - [Info] SSH man-in-the-middle server for 172.20.0.203 listening on 0.0.0.0:10002
2020-11-03 18:08:47.997 - [Debug] [Connection] Attacker connected: 117.50.7.159 | Client Identification: SSH-2.0-Go
2020-11-03 18:08:49.641 - [Debug] [Auth] Attacker 117.50.7.159 trying to authenticate with "none"
2020-11-03 18:08:49.642 - [Debug] [Auth] Attacker authentication error: No authentication method provided
2020-11-03 18:09:03.060 - [Debug] [Connection] Attacker closed the connection
2020-11-03 18:11:34.918 - [Debug] [Connection] Attacker connected: 141.98.81.154 | Client Identification: SSH-2.0-Go
2020-11-03 18:11:35.475 - [Debug] [Auth] Attacker 141.98.81.154 trying to authenticate with "none"
2020-11-03 18:11:35.476 - [Debug] [Auth] Attacker authentication error: No authentication method provided
2020-11-03 18:11:35.688 - [Debug] [Auth] Attacker 141.98.81.154 trying to authenticate with "password"
2020-11-03 18:11:35.689 - [Debug] [Auto Access] Attacker: 141.98.81.154, Threshold: 1, Attempts: 1
2020-11-03 18:11:35.689 - [Debug] [Auto Access] Compromising the honeypot
2020-11-03 18:11:35.689 - [Debug] [Auto Access] Adding the following credentials: "admin:admin"
2020-11-03 18:11:35.895 - [Debug] [Auth] Valid credentials - Password Authentication
2020-11-03 18:11:35.895 - [Debug] [LXC] Attempting to connect to the honeypot: 172.20.0.203
2020-11-03 18:11:51.225 - [Debug] [LXC-Auth] Attacker authenticated and is inside container
2020-11-03 18:11:51.648 - [Debug] [Connection] Attacker closed connection
2020-11-03 18:11:51.808 - [Debug] [LXC] Container's OpenSSH server ended connection
2020-11-03 18:11:51.809 - [Debug] [LXC] Container's OpenSSH server closed connection
2020-11-03 18:13:11.317 - [Debug] [Connection] Attacker connected: 141.98.81.154 | Client Identification: SSH-2.0-Go
2020-11-03 18:13:11.913 - [Debug] [Auth] Attacker 141.98.81.154 trying to authenticate with "none"
2020-11-03 18:13:11.913 - [Debug] [Auth] Attacker authentication error: No authentication method provided
2020-11-03 18:13:12.133 - [Debug] [Auth] Attacker 141.98.81.154 trying to authenticate with "password"
2020-11-03 18:13:12.134 - [Debug] [Auth] Attacker authentication error: Invalid credentials - Password Authentication Failure
2020-11-03 18:13:12.135 - [Debug] [Auth] Attacker: 141.98.81.154 has so far made 1 attempts. Remaining: 5 attempts
2020-11-03 18:13:12.353 - [Debug] [Connection] Attacker closed the connection
2020-11-03 18:16:52.539 - [Info] Exiting...
2020-11-03 18:16:52.540 - [Debug] Cleaning up...
